### ISSUE TITLE
gaybeeg.info

### ISSUE THREAD

**Issue by hatplay** [reek/anti-adblock-killer#3107]
http://gaybeeg.info

![image](https://cloud.githubusercontent.com/assets/27808164/25224716/d0e0cc2c-25f1-11e7-918e-dc492a1a6c19.png)

Chrome
None
uBlock Origin
EasyList + AakList + Adblock Warning Removal List

**Comment by uBlock-user** [comment]
add 

```
gaybeeg.info##script:inject(noeval.js)
gaybeeg.info##.bugme
```

**Comment by hatplay** [comment]
Solved! Thank you! :)

**Comment by hatplay** [comment]
Hi @uBlock-user , there comes another issue. The anti-adblocker prompt no longer exists, but the download links also become invalid when the uBlock is on. Here is the screenshot. Thanks in advance!

![image](https://cloud.githubusercontent.com/assets/27808164/25280753/990d6480-26dd-11e7-93d0-f4bdea648249.png)

**Comment by uBlock-user** [comment]
Not an issue but a personal saobotage by the website itself, cannot be fixed by uBO with current scriptlets. You will need to manipulate via JS injection specifically like this - `a.readOnly("_blocked", function ( ) { });`

**Comment by hatplay** [comment]
Thank you! @uBlock-user But could you give more details on JS injection? I am a newbie on coding...

**Comment by uBlock-user** [comment]
@jspenguin2017 Adding `a.readOnly("_blocked", function ( ) { });` is the solution.

You already have AAK script, remove it and install Adblock Protector Script once this solution is added there.

**Comment by hatplay** [comment]
Thank you, @uBlock-user ! But how to install Adblock Protector Script? I didn't find it on uBlock's 3rd-party-filters. I tried adding  `a.readOnly("_blocked", function ( ) { });` to My filters, but it didn't work. Or do I need to install plugin Tampermonkey to add the script?

**Comment by uBlock-user** [comment]
Ofcourse you will need a script host like TM, once you install TM, then install the script.

**Comment by phineasmax** [comment]
Tried to add
gaybeeg.info##script:inject(noeval.js)
gaybeeg.info##.bugme
to adblock plus, but it's still detected.
Firefox
None
Adblock Plus
EasyList + AakList.

**Comment by uBlock-user** [comment]
Eh...Those filters are for uBO only, not supported by ABP.

**Comment by jspenguin2017** [comment]
@uBlock-user Thanks, I'll have a look.

**Comment by jspenguin2017** [comment]
@uBlock-user That didn't quite worked for me. Remove the `noeval.js` and add this: 
```JavaScript
if (a.domCmp(["gaybeeg.info"])) {
    //NSFW!
    a.readOnly("eval", function () {
        a.$(".download a.button").each(function (i, el) {
            a.$(el).removeClass("locked").attr("href", a.$(el).data("href"))
                .removeAttr("data-href");
        });
    });
    a.ready(a.win.eval);
}
```
I'll add it to Script in a minute.

**Comment by jspenguin2017** [comment]
@uBlock-user That didn't quite work for me... I think this should work: 
```JavaScript
if (a.domCmp(["gaybeeg.info"])) {
    //NSFW!
    a.readOnly("eval", function () { });
    a.ready(function () {
        a.$(".download a.button").each(function (i, el) {
            a.$(el).removeClass("locked").attr("href", a.$(el).data("href"))
                .removeAttr("data-href");
        });
    });
}
```

Update: I guess if you don't use `noeval` and lock `_blocked` to `0` would do the job too... But I'd like to not let a pile of garbage to execute if I can...

**Comment by uBlock-user** [comment]
Works for me with that one line itself, no anti adblock notice and the download button is green instead of inactive and grey.

**Comment by jspenguin2017** [comment]
@uBlock-user OK, if you remove `noeval`, then locking `_blocked` does work. I don't really like letting them execute the detection code, but OK, what ever works. 

As a side note, lock variable to a simple type will be faster. In this case, it would be integer `0`.

**Comment by jspenguin2017** [comment]
OK, added to Script and List 👍 Thanks for the help.

**Comment by phineasmax** [comment]
Wating for ABP :(

**Comment by uBlock-user** [comment]
Don't bother, there's none. You will have to whitelist some third-party connections to make it work.

**Comment by hatplay** [comment]
Oh no. The Script and List are not working AGAIN. They worked yesterday. @uBlock-user @jspenguin2017

**Comment by phineasmax** [comment]
Hello. Tried uBlock and It works perfectly but the download link is inactive and gray.
Also tried to add script to Greasemonkey but didn't work.

**Comment by jspenguin2017** [comment]
Hum... You are right, it broke... I knew locking `_blocked` won't last... I'll fix it. 
Update: Fixed. Update Script and List and let me know if it works. 
@phineasmax Script is not compatible with Greasemonkey. Please use Tampermonkey instead.

**Comment by uBlock-user** [comment]
@hatplay Yeah the webmaster patched it, force an update to the script, should be patched again.

**Comment by uBlock-user** [comment]
Add `a.readOnly("_blocked", function ( ) { });` to your solution too, that takes care of the saotaged download link.

**Comment by jspenguin2017** [comment]
@uBlock-user Can't reproduce. The download is filled with Adfly-like garbage but they aren't replaced.
